File: src/test/scala/cern/colt/matrix/tdouble/impl/SparseRCDoubleMatrix2DViewTest.scala
package cern.colt.matrix.tdouble.impl

import cern.colt.matrix.MatrixTypes.SparseRCDoubleMatrix2D

class SparseRCDoubleMatrix2DViewTest(arg0: String) extends SparseRCDoubleMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseRCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseRCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseRCDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}

File: src/test/java/cernx/colt/matrix/tdouble/impl/SparseRCDoubleMatrix2DViewTest.java
package cernx.colt.matrix.tdouble.impl;

public class SparseRCDoubleMatrix2DViewTest extends SparseRCDoubleMatrix2DTest {

    public SparseRCDoubleMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseRCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseRCDoubleMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseRCDoubleMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
